langchain4j-mongodb-atlas/src/test/java/dev/langchain4j/store/embedding/mongodb/MongoDbEmbeddingStoreIT.java

```diff
@@ -0,0 +1,67 @@
+package dev.langchain4j.store.embedding.mongodb;
```

Comment:
Renaming class via remove and add wipes all commit history :(
It would be better to use `git mv`

Reply:
Should be resolved via the force push.